lib/types.ts

```
@@ -75,6 +75,10 @@ export type WorkflowSettings = {
      * Exposes access to the kinde localization
      */
     "kinde.localization": {};
+    /**
+     * Required to change the MFA policy
+     */
+    "kinde.mfa": {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)